### DISCUSSION TITLE
Running loki screenshot tests with Storybook for multiple themes

### DISCUSSION THREAD

**Discussion by jkayen** [oblador/loki#423] [Q&A]
Is there an easy way to run all stories with loki when there are multiple themes in a design system, without writing double the stories in with each theme mannualy?

**Comment by cameron-yee** [comment]
@jkayen Did you find a good approach for this?

We got around this by:
1. Wrap every story in a theme [decorator](https://storybook.js.org/docs/react/writing-stories/decorators)
2. Build storybook for each theme (`build-storybook -o ./public/storybook/{THEME}`)
3. Test with Loki for each Storybook build using these options:

- `--reactUri`: `./public/storybook/{THEME}/storybook-static`
- `--output`: `./.loki/{THEME}/current`
- `--reference`: `./.loki/{THEME}/reference`
- `--difference`: `./.loki/{THEME}/difference`

This works well, but it'd be nice if it could be handled in `loki.config.js` somehow instead.
